index.html

```diff
@@ -10,197 +10,208 @@
   </head>
   <body>
     <!--section first with logo menu and first content-->
-    <header class="header container">
-      <div class="header__block block">
-        <h1 class="header__block--logo"><img src="./images/logo.svg" /></h1>
-        <nav class="nav">
-          <ul>
-            <li><a href="#team">The Team</a></li>
-            <li><a href="#pricing">Pricing</a></li>
-            <li><a href="#features">Features</a></li>
-          </ul>
-        </nav>
-      </div>
-      <div class="main__container block">
-        <article class="main__container--article">
-          <h1 class="article__title bold">
-            " Beautiful Free Nova template "
-            <br />
-          </h1>
-          <h2 class="article__text light">
-            A top notch premium quality template for your next web project.
-            <br />
-          </h2>
-          <h3>
-            <button class="article__button">Download FREE!</button>
-          </h3>
-        </article>
-        <aside class="main__container--ads">
-          <img class="ads__img" src="images/screen.png" />
-        </aside>
+    <header class="header">
+      <div class="header_container container">
```

Comment:
Tutaj raczej 2x `_` czyli `header__container`. Jeśli chcesz mieć nazwę z 2 słów to użyj `-` tj. `header-container`